dd-sdk-android/src/test/kotlin/com/datadog/android/core/configuration/ConfigurationBuilderTest.kt

```diff
@@ -102,6 +102,7 @@ internal class ConfigurationBuilderTest {
         assertThat(config.coreConfig).isEqualTo(
             Configuration.Core(
                 needsClearTextHttp = false,
+                verboseDebugInfo = false,
```

Comment:
Can you add a test ensuring the effects of the new `setVerboseDebugInfo()` method here